deployment/ccip/changeset/internal/deploy_home_chain.go

```diff
@@ -409,9 +406,24 @@ func BuildOCR3ConfigForCCIPHome(
 
 		transmittersBytes := make([][]byte, len(transmitters))
 		for i, transmitter := range transmitters {
-			parsed, err2 := common.ParseHexOrString(string(transmitter))
-			if err2 != nil {
-				return nil, err2
+			// TODO: this should just use the addresscodec
```

Comment:
Can we not create the address codec here? `deployment` is dependent on `chainlink` AFAIK